tensorflow-whl/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu18.04 AS nvidia
-FROM gcr.io/deeplearning-platform-release/base-cpu:m61
+FROM gcr.io/deeplearning-platform-release/base-cpu:m64
```

Comment:
To build a new TF whl, your branch must have the `-whl` suffix. However, I don't think we need a new wheel because the output whl won't be different with this base image (same OS version). If the TF tests pass, then it should be all good.

Reply:
Okay I'll revert that change thanks.

Reply:
Actually, don't revert the change. It is good. The next time we build an image for a new TF version it will be good to use the same base image.